src/components/Forms/TextFieldHint.js

```diff
@@ -35,17 +37,22 @@ class TextFieldHint extends Component {
     show: PropTypes.bool,
     /** Override styles */
     style: PropTypes.object,
+    /** A uniq id */
+    inputId: PropTypes.string,
   }
 
   static defaultProps = {
     disabled: false,
   }
 
   render() {
-    const { disabled, show, style, text } = this.props
+    const { disabled, show, style, text, inputId } = this.props
 
     return (
-      <div style={[styles.root, style, disabled && styles.disabled, show && styles.show]}>
+      <div
+        id={`hint_${inputId}`}
```

Comment:
more of a nit - since you have to define the hint_<x> class in the parent component anyways, you can pass in the full `hint_${inputId}` string instead of just `inputId`.